genny/resource/templates/standard/actions/resource-name.go.tmpl

```diff
@@ -11,8 +11,8 @@ type {{.opts.Name.Resource}}Resource struct{
 }
 
 {{ range $a := .actions }}
-// {{$a.String}} default implementation.
-func (v {{$.opts.Name.Resource}}Resource) {{$a.String}}(c buffalo.Context) error {
+// {{$a.Pascalize}} default implementation.
+func (v {{$.opts.Name.Resource}}Resource) {{$a.Pascalize}}(c buffalo.Context) error {
   return c.Render(http.StatusOK, r.String("{{$.opts.Model.Proper}}#{{$a.String}}"))
```

Comment:
Don't forget about the "$a.String" in this one:) I believe this should also be "$a.Pascalize"?

Reply:
Covered. Thanks!